solutions/usaco-711.mdx

```diff
@@ -0,0 +1,46 @@
+---
+id: usaco-711
+source: USACO Bronze 2017 Third Contest
```

Comment:
> source: USACO Bronze 2017 Third Contest

not in the same format as https://usaco.guide/general/adding-solution?lang=cpp#steps